src/MainPlayer.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 
-
-class MainPlayer extends React.Component {
-
-  render() {
-    var videoId = 'kIRab-Rn_Ik';
-    if (this.props.vidInfo) {
-      videoId = this.props.vidInfo
-    }
-
-    return (<div className='mainplayer'>
-      <iframe width="560" height="315" src={"https://www.youtube.com/embed/" + videoId} frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
+const MainPlayer = ({
```

Comment:
I thought using a function compnent would be better suited for this component since it's not managing React state.

Reply:
Class components without state have been converted to functional components